examples/caching/weather.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios'
```

Comment:
there is no need to introduce an extra dependency. you can use `https` module:

```js
import https from 'node:https'

https
  .get(`https://reqres.in/api/users`, resp => {
    let data = "";

    // A chunk of data has been recieved.
    resp.on("data", chunk => {
      data += chunk;
    });

    // The whole response has been received. Print out the result.
    resp.on("end", () => {
      let url = JSON.parse(data).message;
      console.log(url);      
    });
  })
  .on("error", err => {
    console.log("Error: " + err.message);
  })
```

Reply:
Or just use global `fetch` https://nodejs.org/api/globals.html#fetch

Reply:
For some reason fetch was not available. Probably because my node version was old.